app/LAMPAPI/Contact.php

```diff
@@ -1,9 +1,13 @@
 <?php
-require_once('../config/config.php');
 
 // Setting global variable to use database connection for different operations
 global $conn;
 
+$dbServer = getenv('DB_SERVER');
+$dbUsername = getenv('DB_USERNAME');
+$dbPassword = getenv('DB_PASSWORD');
+$dbName = getenv('DB_NAME');
+
```

Comment:
@Osa-nag00, @Shi-morrison
Adding these lines allow us to get the db info from our .env file, removing the need for a config.php file

Reply:
Cool works for me!